testsuite/tests/kuadrant/identical_hostname/conftest.py

```diff
@@ -0,0 +1,27 @@
+"""Conftest for "identical hostname" tests"""
+
+import pytest
+
+from testsuite.gateway import GatewayRoute
+from testsuite.gateway.envoy.route import EnvoyVirtualRoute
+from testsuite.gateway.gateway_api.route import HTTPRoute
+
+
+@pytest.fixture(scope="module", autouse=True)
+def route2(request, kuadrant, gateway, blame, hostname, backend, module_label) -> GatewayRoute:
+    """HTTPRoute object serving as a 2nd route declaring identical hostname but different path"""
+    if kuadrant:
+        route = HTTPRoute.create_instance(gateway.openshift, blame("route"), gateway, {"app": module_label})
+    else:
+        route = EnvoyVirtualRoute.create_instance(gateway.openshift, blame("route"), gateway)
```

Comment:
Not needed. It will never work with pure envoy

Reply:
Sure, will refactor. Thanks!

Reply:
Done